js/script.js

```diff
@@ -4,55 +4,69 @@ const titleInput = bookForm.title;
 const authorInput = bookForm.author;
 const books = JSON.parse(localStorage.getItem('books')) || [];
 
-const addBook = (title, author) => {
-  books.push({
+class AwesomeBooks {
+  // constructor() {
+  //   this.addBook = this.addBook.bind(this);
+  //   this.createBookElement = this.createBookElement.bind(this);
+  // }
```

Comment:
- [ ] Please remove the commented code from the files, To make your project code clean and professional.